src/Fable.Core/Fable.Core.Types.fs

```diff
@@ -15,8 +67,68 @@ type CaseRules =
     /// FooBar -> foobar
     | LowerAll = 5
 
+/// <summary>
 /// Used on interfaces to mangle member names. This allows overloading and prevents conflicts
 /// other interfaces, but will make interop with native code more difficult.
+/// </summary>
+/// <remarks>
+/// <para>
+/// If you are not planning to use an interface to interact with JS and want to have overloaded
+/// members, you can decorate the interface declaration with the <c>Mangle</c> attribute.
+/// </para>
+/// <para>
+/// Interface coming from .NET BCL (like System.Collections.IEnumerator) are mangled by default.
+/// </para>
+/// <para>
+/// For example the following code:
+/// </para>
+/// <code lang="fsharp">
+/// type IRenderer =
+///     abstract Render : unit -> string
+///     abstract Render : indentation : int -> string
+///
+/// type Renderer() =
+///     interface IRenderer with
+///         member this.Render() =
+///             failwith "Not implemented"
+///
+///         member this.Render(indentation) =
+///             failwith "Not implemented"
+/// </code>
+/// <para>
+/// Generates invalid JavaScript code. Note the two methods named <c>Render</c> in the same class.
+/// </para>
+/// <code lang="js">
+/// export class Renderer {
+///     constructor() {
+///     }
+///     Render() {
+///         throw new Error("Not implemented");
+///     }
+///     Render(indentation) {
+///         throw new Error("Not implemented");
+///     }
+/// }
+/// </code>
```

Comment:
One potential issue I see with including target code, is that some of the code is not JavaScript specific.

Adding a version of the code for all the target don't seems do-able because of the work needed, and the tooltip becoming way too big ?

Should we consider linking to an external resource?

Reply:
> > Make it 'pseudocode'?
> 
> I don't think it would work, but the code between the language perhaps be quite different.

Have a look at the recent commits to see how I've tried to provide examples using 'pseudocode'. If you don't reckon that will work, then we'll have to do without target example code and refer to the docs as you say.

I've just realised another annoying difference between Ionide and Rider is that the `seealso` href is not rendered as a link as it is in Rider.